packages/elements/src/internals/machines/sign-in/utils/starting-factors.ts

```diff
@@ -1,9 +1,10 @@
 // These utilities are ported from: packages/clerk-js/src/ui/components/SignIn/utils.ts
 // They should be functionally identical.
+import { isWebAuthnSupported } from '@clerk/clerk-js/src/utils/passkeys';
```

Comment:
Is this even allowed? Do we need to move this to shared ?

Reply:
I don't think this will work in the built package, so you have two options:

1. Copy/paste into this file
2. Move this to shared and use from there

Reply:
Shame, I will add it to shared